ui/src/app/components/demographic-survey-v2.tsx

```diff
@@ -990,10 +1001,13 @@ export const DemographicSurvey = fp.flow(withProfileErrorModal)(
               in the preceding questions that we may want to consider including
               in future surveys?
             </div>
-            <TextInput
+            <TextAreaWithLengthValidationMessage
+              id='survey-comments'
               onChange={(value) => onUpdate('surveyComments', value)}
+              initialText={survey.surveyComments || ''}
               value={survey.surveyComments || ''}
```

Comment:
this component doesn't have a `value` so it fails to compile.

It looks like this can simply be removed because `initialText` and `onChange` are already set to do the right thing.